Week3/homework/step2-2.js

```diff
@@ -3,17 +3,24 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
```

Comment:
I overlooked this on my first review. You were supposed to create this `numbers` array first (with a loop) and then use it in your second loop.

Reply:
Done.